README.md

```diff
@@ -28,12 +31,16 @@ variable.
 Usage: cow-fee [options]
 
 Options:
-  --network <network>         (choices: "mainnet", "gnosis")
+  --network <network>                           (choices: "mainnet", "gnosis")
   --rpc-url <rpc-url>
-  --min-value <min-value>    Minimum USD value of token to swap (default: 1000)
-  --max-orders <max-orders>  Maximum number of orders to place in single drip call (default: 250)
-  --min-out <min-out>        Minimum amount of to-token to receive per swap (default: 0.02)
-  -h, --help                 display help for command
+  --min-value <min-value>                      Minimum USD value of token to swap (default: 1000)
+  --max-orders <max-orders>                    Maximum number of orders to place in single drip call (default: 250)
+  --min-out <min-out>                          Minimum amount of to-token to receive per swap (default: 0.02)
```

Comment:
min-value/min-out seem equivalent to me (just in different currencies). Can we combine into one for operational simplicity? I actually a think a minOut amount (in native token) is sufficient and can save us some more chain specific USD conversion logic.